fuse/data/utils/collates.py

```diff
@@ -80,10 +80,10 @@ def __call__(self, samples: List[Dict]) -> Dict:
 
             try:
                 # collect values into a list
-                collected_values, has_error = self._collect_values_to_list(samples, key)
+                collected_values, has_error, has_missing_values = self._collect_values_to_list(samples, key)
 
                 # batch values
-                self._batch_dispatch(batch_dict, samples, key, has_error, collected_values)
+                self._batch_dispatch(batch_dict, samples, key, has_error or has_missing_values, collected_values)
```

Comment:
The signature of `_batch_dispatch()` contains only `has_error` has a parameter. Should we add something to mention `has_missing_values` ?

Reply:
The signature is located in line 94